### PR TITLE
First Card Component PR

### PR DESCRIPTION
**Pull request by 4nth0ny1**
created search results page at localhost:3000/search
created creator page /creator with courses section and creator section and review section
using fake data 
it may be feasible to use TextImage down the road, but for now i elected to create new components.

### CODE REVIEW

**Comment by 4nth0ny1** [comment]
Hold off on this. I forgot that this PR is supposed to be all the cards on the site.

**Comment by 4nth0ny1** [comment]
this is getting too big for one PR. I'm going to make another Cards one after this is approved.

**Review by cruzma — APPROVED** [review]

**Review by zxcv3 — APPROVED** [review]
It seems to be ok

**Pull request merged**
